cmake/OpenDDSConfig.cmake

```diff
@@ -222,10 +257,14 @@ set(OPENDDS_RTPS_UDP_DEPS
   OpenDDS::Rtps
 )
 
-set(OPENDDS_SECURITY_DEPS
-  OpenDDS::Rtps
-  ACE::XML_Utils
-)
+if(OPENDDS_SECURITY)
+  set(OPENDDS_SECURITY_DEPS
+    OpenDDS::Rtps
+    ACE::XML_Utils
+    OpenSSL::SSL
```

Comment:
SSL shouldn't actually be a dependency, only Crypto.  Not sure if we'd need to change something in the OpenDDS build itself to reflect this.  At least ldd only shows libcrypto.

Reply:
I'm just trying to match what MPC is doing.